lib/token-request.js

```diff
@@ -321,7 +321,7 @@ TokenRequest.prototype._getTokenUsernamePasswordFederated = function(username, p
       return;
     }
 
-    this._performUsernamePasswordForAccessTokenExchange(this._userRealm.federationActiveAuthUrl, username, password, callback);
+    this._performUsernamePasswordForAccessTokenExchange(this._userRealm.federationActiveAuthUrl, constants.WSTrustVersion.UNDEFINED, username, password, callback);
```

Comment:
But this is under the if case where the federationMetadataUrl is null, it won't go through the mex parsing. If it won't even go through the mex parsing, should we check the wstrust version?